### ISSUE TITLE
Update tarallo_login() method for compatibility with v2

### ISSUE THREAD

**Issue by e-caste** [WEEE-Open/turbofresa#22]

**Comment by lvps** [comment]
That method isn't even needed anymore, use pytarallo, and the concept of "login" doesn't even exist anymore for APIs: pass the token to pytarallo.Tarallo() and call whatever method you need (`get_item`, etc...)

**Comment by lvps** [comment]
Minimalist example:

```python
from pytarallo.Tarallo import Tarallo

TARALLO_URL = "http://127.0.0.1:8080"
TARALLO_TOKEN = "generate a token from the GUI and place it here"

t = Tarallo(TARALLO_URL, TARALLO_TOKEN)
t.get_item("HDD123")  # or whatever method you need
```

The UPPERCASE variables should become environment variables or lines in a configuration file or something some day in the future, but for now they can be hardcoded

**Comment by lvps** [comment]
Specifically, if you need to search for an item given its serial number, the relevant part is:

```python
t = Tarallo(TARALLO_URL, TARALLO_TOKEN)  # as above, you only need to do this once
items = t.get_codes_by_feature("sn", serial_number)

if len(items) != 1:
    raise SomeError()  # or print an error message
item_code = items[0]
```

"sn" is the feature name for serial numbers. You could also use WWN or other features, but this one is the easiest and most universal one for the job.

`items` is a list of items, e.g. `['H150', 'S164']`, even though it should hopefully contain only a single item, duplicate serial numbers are usually a mistake. If the list is empty, no items with that serial number have been found.

So, if you need the entire item, e.g. you need to check its location or other features:

```python
item = t.get_item(item_code)
```

Or if you want to edit the item:

```python
features = {
    "erased": "yes"
}
t.update_features(item_code, features)
```

This will mark the hard disk as erased.

Or maybe you want to move it:

```python
t.move(item_code, "code of the location")
```

**Comment by lvps** [comment]
One final note: the `get_codes_by_feature` method is available from pytarallo 1.2 onwards, which I released about 5 minutes ago.

Run `pipenv update` in the turbofresa directory to download the latest version and you're good to go!

**Comment by e-caste** [comment]
Closed with PR #24 by @TsumiYaku
